cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,21 +1,33 @@
+import {
+  ItemPage, LoginPage, ProductListPage,
+  ShoppingCartPage, CheckoutCompletePage,
+  InformationPage, OverViewPage
+} from '../page'
+
+const loginPage = new LoginPage()
+const productList = new ProductListPage()
+const shoppingCart = new ShoppingCartPage()
+const itemPage = new ItemPage()
+const checkoutComplete = new CheckoutCompletePage()
+const informationPage = new InformationPage()
+const overViewe = new OverViewPage()
```

Comment:
a better name for this object can be `overviewPage`

Reply:
Fixed